package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-select-event",
-  "version": "4.1.4",
+  "version": "4.2.0",
```

Comment:
Would you mind reverting this?
It'll get done when I publish the package :slightly_smiling_face: